README.md

```diff
@@ -1,15 +1,16 @@
 <p align="center">
-  <a href="https://terraform.io">
-    <img src="https://cdn.rawgit.com/hashicorp/terraform-website/master/content/source/assets/images/logo-hashicorp.svg" alt="Terraform" width="330px">
-  </a>
-  <img width="30px"/>
-  <a href="https://secrethub.io">
-    <img src="https://secrethub.io/img/secrethub-logo.svg" alt="SecretHub" width="360px">
+  <img src="https://secrethub.io/img/integrations/terraform/github-banner.png?v1" alt="Terraform + SecretHub" width="390">
   </a>
```

Comment:
Copy-paste of https://github.com/secrethub/secrethub-circleci-orb/pull/30 ?

Reply:
Yes, I'll remove it here too.